client/state/reader/follows/reducer.js

```diff
@@ -158,6 +159,25 @@ export const items = createReducer(
 			);
 			return merge( {}, state, keyedNewFollows );
 		},
+		[ READER_SITE_REQUEST_SUCCESS ]: ( state, action ) => {
+			const incomingSite = action.payload;
+			if ( ! incomingSite || ! incomingSite.is_following ) {
+				return state;
+			}
+			const urlKey = prepareComparableUrl( incomingSite.feed_URL );
```

Comment:
there's a chance the site won't have a feed_URL... if this is a pure email subscription, there may be no corresponding feed for the site.

Reply:
if so, it'd also be a good test case to add

Reply:
Good shout. I'll drop if there's no feed URL and add a test.